packages/gp2-components/src/templates/OutputForm.tsx

```diff
@@ -253,12 +260,10 @@ const OutputForm: React.FC<OutputFormType> = ({
                 ></Markdown>
               }
             />
-            {!['Training Materials', 'Procedural Form'].includes(
-              documentType,
-            ) ? (
+            {!DOC_TYPES_GP2_SUPPORTED_NOT_REQUIRED.includes(documentType) ? (
               <LabeledRadioButtonGroup<gp2Model.DecisionOption>
                 title="Has this output been supported by GP2?"
-                subtitle="(required)"
+                subtitle="(optional)"
```

Comment:
This field should be filled when it is shown, so for the user, we should say it's required, as it even has a default value "Don't Know". If it was optional, we would have nothing selected as default. does it make sense?

Reply:
Ok! it makes sense

Reply:
committed here [change gp2 supported subtitle to required again](https://github.com/yldio/asap-hub/pull/3728/commits/547256297716ad63e79e9f48b7c729b9f9c216c3)